Makefile

```diff
@@ -0,0 +1,9 @@
+# simple makefile to simplify repetitive build env management tasks
+
+test-coverage: python -m pytest test -m "not notebooks" --ignore=test/perf --ignore=test/install --cov=fairlearn --cov-report=xml --cov-report=html
+
+test-unit: python -m pytest ./test/unit
```

Comment:
Somewhere in your scripts you're cleaning the build directories before running the build.

Usually, when we call a make command, the expectation is that it should only run the changed files.

If the user wants to start from scratch, they do `make clean`, and the `clean` command in the makefile takes care of removing all temp and build directories.

There are other common commands that we can add, but I can test and add them once you get this one in, in a separate PR.

Reply:
That would be fantastic. I guess I do that with `git clean -xdf` manually, but some of our scripts make sure that we start with clean directories, that's true

Reply:
Can we put additional changes off, and you'll just adjust/enhance this file as and when you need it? I think you're the only user of this right now.

Reply:
yes, I can enhance it later.